docs/USAGE-GUIDE.md

```diff
@@ -1388,6 +1388,30 @@ Return the certificate that was used during the TLS handshake.
 This function returns NULL if the certificate selection phase of the handshake has not completed
  or if a certificate was not requested by the peer.
 
+### s2n\_get\_cert\_chain\_length
+
+```c
+int s2n_get_cert_chain_length(struct s2n_cert_chain_and_key *chain_and_key, uint32_t *cert_length);
+```
+
+**s2n_get_cert_chain_length** gets the length of the certificate chain `chain_and_key`. If the certificate chain `chain_and_key` is NULL an error is thrown. If the certificate chain contains only one ceritificate the length returned is equal to one. 
+
```

Comment:
Why would you specify the behavior for a single certificate?

Reply:
Because the cert_idx ranges from `0`  to `cert_length - 1`, which would mean the `0` index represents a cert chain length of `1`.  I was trying to make it clear that the `cert_length` does not match the index  values here by stating that a certificate chain with a single cert in it would return a cert length equal to one .

Reply:
I think you cover this by explaining that the index starts at 0 in the other API. Mentioning that a cert chain one certificate long has a length of 1 makes me think I'm missing something tricky :)